configs/templates/unified-dev/spack.yaml

```diff
@@ -25,11 +25,13 @@ spack:
       #- upp-env
       #- ww3-env
 
-      # Additional esmf/mapl tags
-      #- esmf@8.4.2~debug+external-parallelio
-      #- mapl@2.35.2~debug ^esmf@8.4.2~debug+external-parallelio
+      # Various fms tags (list all to avoid duplicate packages)
+      - fms@release-jcsda
+      - fms@2023.01
+      - fms@2023.02
```

Comment:
3 versions of fms moving forward? just wanted to double check. i saw this https://github.com/ufs-community/ufs-weather-model/issues/1874, so assuming we retain 2023.01 for a while until all apps move to 2023.02?